src/web/arr/trove/chart.arr

```diff
@@ -2449,4 +2560,5 @@ from-list = {
   freq-bar-chart: freq-bar-chart-from-list,
   labeled-box-plot: labeled-box-plot-from-list,
   box-plot: box-plot-from-list,
+  residual-chart: residual-chart-from-list,
```

Comment:
I think you've already renamed this -- now I'm wondering if Git is just showing me an old version of the file?